### ISSUE TITLE
[Sybil Report]

### ISSUE THREAD

**Issue by ceMaBenTengHa** [LayerZero-Labs/sybil-report#6129]
# Reported Addresses
0x8b4e06050dcfa9ba4eee6d57da7f24ba403c188a
0x6f845814eb854e564423aaaf7c7fcd80842d9d5a
0xb10ac613a083c1164530e53e4eaf75786fab664e
0x94d39f7e0c24651c784e4471444bba973a2b5bae
0x2dbc1ef60c465ad388d39c974b2559d1b350d561
0x591801872032da7952fdc4383117d7d927c6547c
0xe65ef1ba40302651df46954c6c719e2f8af82e7c
0x5bfe586807ea263462022605f9c9dcd64ea07b31
0x8ed8255444f550a4ea886b3f51e328da68c7be15
0xc42feb5d376ff153ed57c491345293991b6270ba
0xf1e4530ad1e89b403e1f229b9029dadbbce413fc
0x663b1874079142b1c25951133c8e7d1be10935b6
0x57f9d09f8a0a9e627f945e679fa67b79fc803e11
0xcdcefe8ba11a230c6081d03cba9af0763f9fd6e3
0x1268b9e3c5d22a885822432e9b750fad80ef8388
0x84bbe67ee7642f92b564f4fa32bff17d5fd333cc
0x229727239289d05deed1901a986d4d703c929cd1
0x3705457cb5c8f72c401464102f39f202c5e330cd
0x52f588677672651d54d88bfd0320aa24ac3dfd52
0x798fd8189307b4861a0cbf98f0694ac001466fcc
0xa6f0a6e251d932e7f4d56439016db08fa9a9df6e
0xaed6ecbd4ac02e9632f163339032e9d2dc9a78c0
0xf74fc2b37b45cf3eb6499ff6d53dbd8d421375a1
0x4471a4da63b76ebc8108addd7478a4b2cc7d6c5a
0x7308ed37df3a98e40bb0aa35d72e04df44e2068e
0x48454926836dc5ee1ed9e5707cbf943131340b40
0xac6e823a9075f698aa709c162e49e56ff8b93610
0x6fe846ae799b36bad8ac4da8741b60dfc6c3186f
0x0ce81164410191d510f27f2778b0fb7811787e9d
0xfad1ec9bb8592c087cfdc2e7309d618182570e4f
0x913fe09667228362555fe1f57bc2ffe294ab982c
0x3dce0f83c897d8e10347edd54e6a76a11cd99ece
0x29f8d769c364e26032427ff1b0360b2d218c70f7
0x692842eceeba51df6ce0bcda867e3dfc455f9961
0x0ba26e6cbe72648190a27a65c0f0027eaa3e078c
0xae79243fc8f57d82faea5e0197e910ed27114df2
0x3fa8075dca51a1990b90222b0c3bca8fe840c53d
0x58eacfb617d86d4460b77e405d2fc2ece1e50983
0x1e73abfaa67f62b958c02a6d225f450aec106e46
0x5e782c765166872b9b9888a6dd8b619e73f22b60
0x6f907a66b3878d0dab726888ab995e3a1ec78c7e
0x1b4ec5b3fc9c22da235665bffab3e7dd30d94447
0x1db8f362b78d5f47ee4c48030f001ed65809d9eb
0xc44c27f65dc7d388c9f1d23474943d9e005674a7
0xfceadc52b3592ddf97c432f0bf401f24879f576b
0x4aff13dc5a7d50c90f4c21be5121eee1eea2dab9
0xab1fd55e355a1e53b5968bbaf978fa442ac5a509
0xce64742e2a61884d1c8b754a8e866566728b5fa4
0xf3694f9bb3a2c874085e91094b871d88b7bb4c30
0xb90aaf5dfd1b3cbfeaae50915785cb5afc2ea7cc
0x5036e6f312a92747852ed3222576db70cda5d523
0xd3969363815794281642369ece043af7e4632c12
0x50eb88cceea33955e53967af2fe67159aea18632
0x9e15ceba646606f65f1447e868b4fd61cf20efde
0x61a310d06bd2bfd92ee69842d0cd017d69091ab0
0xa0738f5819ce4ada6d06ce77250e04f0030eae04
0x85658c06cdd763631715c93a53cb5df360879fe2
0x2a6635f827c91f5177777b3a31d78848e42d14b2
0xf306657ad975e052412a6ceef0799690a8a4b9c2
0x86767000061109adf96c36a8d1666f1399600514
0xdda6e24cdfd75f6beca7f8e51699d229f1dd5a02
0x7f5245b537af9c79af38290957c93abe55d015d8
0x1e39b6b012d5689a4a7d56ae596accb190f36bb9
0xc22c40e65d894c749ef415ea354f736f42ffc7c1
0x6e1f273b4d9194aa57cc4eea521e118fa48f2fde
0xcf29ab716cd8593dbc923e9637393c2188beb315
0x7504a13a114cd625c439aa8808327f1719351208
0x4a5e29112173a41d244b57d83c2c931069ad9f46
0x4bfd327e960a2596d920df621e8d9a2ca9f45518
0xd99427e218de011c2c2904a3c0d467f699f0544b
0x37bdc661d8bc6774aebab131b572243a684d67c5
0x142c9ed7597fae27372164afece283c6c0de27ff
0xe3bfbaa352880e71ac0a4b0ae3f05145efd5be40
0x95e365b892ef270985025392dd0f5b4405865f5e
0xb66731991075329c63eb248f523a45e09d80a67b
0x476a153a1724b63a7bed2cd109a14f5371be3268
0xcd67fe8a35a68f11106e5413936d54073f8ba415
0x989c3fe74a7eb360522a57f76dcc1024dfd9574d
0x7d06a03cf139c2be3eb3531e5c6259d46f61ccfd
 

# Description

Batch operation malicious brushing the same path .
mostly 70% merkly app

# Detailed Methodology & Walkthrough
Batch operation malicious brushing
# Reward Address (If Eligible)
0x9cec91f3D45Ae3ad8E188F5B01d5c5c23500160C

**Comment by daniil45korzich** [comment]
Yes, I'm also in this chat and I know this guy. He makes over 20 accounts, I'm sure that's all you could find, bro? I wanted to post these wallets here, but it’s a pity I didn’t have time.
